ui/shared/Revision.jsx

```diff
@@ -62,8 +62,10 @@ export class Revision extends React.PureComponent {
     const {
       revision: { comments, author, revision },
       repo,
+      bugSummaryMap,
     } = this.props;
     const comment = comments.split('\n')[0];
+    const bugMatches = comment.match(/-- ([0-9]+)|bug.([0-9]+)/gi);
```

Comment:
I also noticed that the clipboard icons are getting left behind as one hovers over the bug tooltips.  I think there's just too much happening there.  And we don't need that clipboard icon when you hover for the bug summary.  So a fix would be to move the `onMouseEnter` and `onMouseLeave` event handlers away from the `<Row>` element and down into the `<span className="pr-1 text-nowrap">` element.  Then the clipboad icon only shows when you hover the commit sha, which makes more sense anyway.  :)  Would you make this change?

Reply:
Perfect!  :)